client/src/components/custom-dropdown.js

```diff
@@ -1,33 +1,104 @@
 import React from 'react';
 import { Container, Dropdown } from 'react-bootstrap';
-import MyDatePicker from './date-picker';
+import DatePicker from 'react-datepicker';
+import DropdownToggle from 'react-bootstrap/DropdownToggle';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
 
 class DropdownWithDate extends React.Component {
+  constructor(props) {
+    super(props);
+    const date = new Date();
+    this.state = {
+      startDate: date,
+      endDate: date,
+    };
+    this.handleOnApplyClick = this.handleOnApplyClick.bind(this);
+  }
+
+  handleOnApplyClick() {
+    const { startDate, endDate } = this.state;
+    const { oneDate, rangeDate } = this.props;
+
+    let path = `${rangeDate + this.formatDate(startDate)}&to=${this.formatDate(endDate)}`;
```

Comment:
You don't need to use plus sign inside template string
`${rangeDate}${this.formatDate(startDate)}&to=${this.formatDate(endDate)}`